### PR TITLE
 Allow owned objects in mock 

### PR DESCRIPTION
**Pull request by janhohenheim**

### CODE REVIEW

File: engine/src/object.rs [hunk 1]
```diff
@@ -8,6 +8,8 @@ use crate::prelude::*;
 mod object_behavior;
 pub use self::object_behavior::*;
 
+
+
 /// An object that is stored in the simulation
```
Comment (jeremystucki) [inline engine/src/object.rs]: TRIGGERED

File: engine/src/object.rs [hunk 2]
```diff
@@ -8,6 +8,8 @@ use crate::prelude::*;
 mod object_behavior;
 pub use self::object_behavior::*;
 
+
```
Comment (tautropfli) [inline engine/src/object.rs]: What?

**Review by jeremystucki — CHANGES_REQUESTED** [review]

**Review by tautropfli — CHANGES_REQUESTED** [review]

**Review by tautropfli — APPROVED** [review]

File: engine/src/object.rs (hunk 2 above)
Comment (janhohenheim) [reply engine/src/object.rs]: Addressed by https://github.com/myelin-ai/engine/pull/31/commits/534493b995482c24cbd4f67be5a11749d29b91cf

File: engine/src/object.rs (hunk 1 above)
Comment (jeremystucki) [reply engine/src/object.rs]: 534493b

**Review by jeremystucki — APPROVED** [review]

**Pull request merged**